.github/workflows/rosa-cluster-auto-delete-on-schedule.yml

```diff
@@ -0,0 +1,16 @@
+name: ROSA Scheduled Delete
+
+on:
+  schedule:
+    - cron: '0 19 * * *' # Runs every day at 7 PM UTC
```

Comment:
Just realised, should we schedule this task only on work days? If I understand correctly this will be triggered also on weekends.

Reply:
good catch, I shall adjust that for both cluster create and delete schedules

probably will use this expression to keep it concise
`- cron: '0 19 * * 1-5'`

Reply:
I accidentally added this comment to the delete workflow. It should probably go mainly to the create workflow.

Reply:
so leave delete for all days but create for only weekdays, is that the final expectation ?

Reply:
Makes sense to me @kami619 even though during the weekends there will be probably no cluster to destroy most of the time, still it causes no harm to run the script.